diff --git a/node_modules/fullpage-vue/dist/fullpage.js b/node_modules/fullpage-vue/dist/fullpage.js
index 0750dc3..3d77e3a 100644
--- a/node_modules/fullpage-vue/dist/fullpage.js
+++ b/node_modules/fullpage-vue/dist/fullpage.js
@@ -106,7 +106,7 @@ var Fullpage = function () {
         this.initScrollDirection();
         this.initEvent(el);
         window.setTimeout(function () {
-            console.log('init', _this);
+            // console.log('init', _this);
             _this.resize();
             var startIndex = _this.opts.start;
             //The first page triggers the animation directly
@@ -119,8 +119,10 @@ var Fullpage = function () {
     createClass(Fullpage, [{
         key: "resize",
         value: function resize() {
-            this.width = this.opts.width || this.el.offsetWidth;
-            this.height = this.opts.height || this.el.offsetHeight;
+            // this.width = this.opts.width || this.el.offsetWidth;
+            // this.height = this.opts.height || this.el.offsetHeight;
+            this.width = this.opts.width || window.innerWidth;
+            this.height = this.opts.height || window.innerHeight;
 
             var i = 0,
                 length = this.pageEles.length,
@@ -292,6 +294,9 @@ var Fullpage = function () {
                 if (!debounce) {
                     return;
                 }
+                if (_this2.opts.dir === 'h') {
+                    return;
+                }
                 debounce = false;
                 clearTimeout(debounceTimer);
                 debounceTimer = setTimeout(function () {
